web/src/utils/constants.ts

```diff
@@ -24,6 +24,7 @@ export const C_HAINES_ROUTE = '/c-haines'
 export const FBP_GO_ROUTE = 'https://sybrand.github.io/'
 export const FIRE_BEHAVIOR_CALC_ROUTE = '/fire-behaviour-calculator'
 export const FIRE_BEHAVIOUR_ADVISORY_ROUTE = '/auto-spatial-advisory'
+export const MORE_CAST_2_ROUTE = '/more-cast-2'
```

Comment:
Minor nit: can we make it `morecast-2` to be consistent with the original `morecast`?

Reply:
I agree with Conor's preference